### PR TITLE
[IMP] Products in subcategories are equivalents and can be excluded

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by bodedra — APPROVED** [review]
LGTM. :+1:

**Review by agyamuta — APPROVED** [review]
LGTM...

File: mrp_bom_equivalent/models/product.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def search_read(self, domain=None, fields=None, offset=0, limit=None,
             category_id = self.browse(
                 self._context.get('nonequivalent_product_id')).categ_id
             domain +=\
-                [('categ_id', '=', category_id.id),
+                [('categ_id', 'child_of', category_id.id),
                  ('id', '!=', self._context.get('nonequivalent_product_id'))]
         order = order or self.priority
```
Comment (LoisRForgeFlow) [inline mrp_bom_equivalent/models/product.py]: Can you check this line? it doesn't make sense for me.

Comment (LoisRForgeFlow) [reply mrp_bom_equivalent/models/product.py]: self.priority on an api.model will return 0. should it be `"priority"`?

**Pull request merged**